ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -1,71 +1,108 @@
-function Get-RDActiveSession {
-    $sessions = Get-RDUserSession | Where-Object { $_.SessionState -eq 'STATE_ACTIVE' -or $_.SessionState -eq 'STATE_IDLE' }
-    $sessions | Select-Object HostServer, UserName, UnifiedSessionId, SessionState | Format-Table
-}
+function Get-sbRDSession {
+    [OutputType('Custom.SB.RDSession')]
+    [cmdletbinding()]
+    param(
+        [Parameter(Mandatory = $false)]
+        [ValidateSet("Active", "Idle", "Connected", "Disconnected", "Any")]
+        [Alias("State")]
+        [string]$SessionState = "Any",
 
-function Get-RDDisconnectedSession {
-    [CmdletBinding()]
-    param (
-        [
-        Parameter(
-            Mandatory = $False,
-            HelpMessage = "Specify minimum number of idle session minutes"
-        )
-        ]
-        [Alias('idle')]
-        [int]$MinIdleTime = 0
+        [Parameter(Mandatory = $false)]
+        [int]$MinimumIdleTime = 0
```

Comment:
Could add validation here, like `[ValidateRange(0,1000)]`.
It's also good practice to include the unit. Is this secs or mins?

Reply:
Very good idea, thank you. When you say "include the unit" - where are you thinking? In the parameter name, or as a comment...?

Reply:
Confirmed variable name as suggestion. "MinimumIdleMins" agreed.